packages/core/admin/scripts/build.js

```diff
@@ -3,16 +3,28 @@
 const path = require('path');
 const webpack = require('webpack');
 const webpackConfig = require('../webpack.config');
+const {
+  getCorePluginsPath,
+  getPluginToInstallPath,
+  createPluginsFile,
+} = require('./create-plugins-file');
+
+const PLUGINS_TO_INSTALL = ['i18n'];
```

Comment:
at the beginning of v4 we will keep users-permissions installed too so you should keep it here